src/WSEvents.cpp

```diff
@@ -1505,6 +1505,25 @@ void WSEvents::OnStudioModeSwitched(bool checked) {
 	broadcastUpdate("StudioModeSwitched", data);
 }
 
+/**
+ * A custom broadcast message was received
+ *
+ * @param {String} `realm` Identifier provided by the sender
+ * @param {Object} `data` User-defined data
+ *
+ * @api events
+ * @name BroadcastCustomMessage
+ * @category general
+ * @since 4.7.0
+ */
+void WSEvents::OnBroadcastCustomMessage(QString realm, OBSDataAutoRelease data) {
```

Comment:
`OBSDataAutoRelease` doesn't increase the reference count but only decreases it. As the calling `WSRequestHandler::BroadcastCustomMessage` function also uses `OBSDataAutoRelease`, this may cause a refcount error.

Either increase it as well by using `OBSData`, or use a plain `obs_data_t*` to not care with reference counting at all (and leave that responsibility to its caller).

Reply:
done